src/main/scala/it/pagopa/interop/purposeprocess/api/impl/ResponseHandlers.scala

```diff
@@ -12,6 +12,17 @@ object ResponseHandlers extends AkkaResponses {
 
   implicit val serviceCode: ServiceCode = ServiceCode("012")
 
+  def retrieveLatestRiskAnalysisConfigurationResponse[T](logMessage: String)(
+    success: T => Route
+  )(result: Try[T])(implicit contexts: Seq[(String, String)], logger: LoggerTakingImplicit[ContextFieldsToLog]): Route =
+    result match {
+      case Success(s)                                           => success(s)
+      case Failure(ex: TenantNotFound)                          => notFound(ex, logMessage)
+      case Failure(ex: RiskAnalysisConfigForTenantKindNotFound) => notFound(ex, logMessage)
+      case Failure(ex: RiskAnalysisConfigLatestVersionNotFound) => notFound(ex, logMessage)
```

Comment:
In these cases we have errors in the configuration, it's ok to return 500 so we get promptly notified
```suggestion
```